apps/humanatlas.io/src/styles.scss

```diff
@@ -1,6 +1,7 @@
 @use 'light-theme';
 @use 'theming';
 @use 'utils';
+@import url('https://fonts.googleapis.com/css2?family=Material+Symbols+Rounded:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200');
```

Comment:
This is not correct. Rounded symbol is already included in the theming. If you need a different optical size apply
```scss
.the-icon-class {
  font-variation-settings:
  'FILL' 0,
  'wght' 300,
  'GRAD' 0,
  'opsz' 24 /* <- Optical size goes here! */
}
```
inside the count-info component instead!